src/nodes/mix-node-setup.md

```diff
@@ -425,30 +425,123 @@ Then reboot your server and restart your mixnode.
+
+You can use either `nym-cli` which can be downloaded from the [release page](https://github.com/nymtech/nym/releases) or compiling `nyxd`.
+
+
+`/path/to/the/release` and run the following on the family head to obtain the signature for the member:
+
+```
+./nym-mixnode sign --id winston-smithnode --text <text>
+```
+
+~~~admonish example collapsible=true title="Console output"
+```
+
+
+
+      _ __  _   _ _ __ ___
+     | '_ \| | | | '_ \ _ \
+     | | | | |_| | | | | | |
+     |_| |_|\__, |_| |_| |_|
+            |___/
+
+             (nym-mixnode - version 1.1.9)
+    
+Signing the text "APxUbCmGp4K9qDzvwVADJFNu8S3JV1AJBw7q6bS5KN9E" using your mixnode's Ed25519 identity key...
+The base58-encoded signature on 'APxUbCmGp4K9qDzvwVADJFNu8S3JV1AJBw7q6bS5KN9E' is: 2ZuCFYU91pvEcgAj6EzU33oozazvsRAoxP7NQHFM6Xy6AkJrzgCZdnsnZYAmxFtqe8Su17KXwpTHQtkVmAnAiV4H
+
+```
+~~~
+
+The `--mnemonic` is going to be the mnemonic of the member wanting to be the head of family: 
+
+
+With `nym-cli`
+
+```
+/nym-cli cosmwasm execute <wallet-address> '{"create_family": {"signature": "<base58-encoded-signature>","family_head": "<text>","owner_signature":"<node owner signature>","label": "<node label>"}}' --mnemonic <mnemonic from node to be the head>
+```
+
+
+With `nyxd`
+
+```
+./nyxd tx wasm execute ${MIXNET-CONTRACT} '{"join_family": {"signature": "<base58-encoded-signature>","family_head": "<text>"}}' --node ${VALIDATOR-ENDPOINT} --from mix1 --chain-id nyx --gas-prices 0.025unym --gas auto --gas-adjustment 1.3 -y -b block
+```
+  
+To get the node owner signature, use:
+
+`./nym-mixnode node-details --id <id>`
+
 ### Joining a Node Family 
 
-Ssh into the vps of the family head and run the following command to obtain the signature for the member. The value is the **identity key** of the mix node which wants to join the family: 
+`/path/to/the/release` and run the following on the family head to obtain the signature for the member:
 
 ```
-./nym-mixnode sign --id mixnode --text 4Yr4qmEHd9sgsuQ83191FR2hD88RfsbMmB4tzhhZWriz
+./nym-mixnode sign --id mixnode --text <text>
 ```
 
-This will return a signature which is going to be used below - in this example it is `3SEjfNcJ5L3cXdvWCdiQNT5DkCFJ2TurK5xsYyEdHH324nAA3bWvKoXmkjU9Xbr9ZyemGDLJ4dmGEHWUwL1LCWKq`. 
+~~~admonish example collapsible=true title="Console output"
+      _ __  _   _ _ __ ___
+     | '_ \| | | | '_ \ _ \
+     | | | | |_| | | | | | |
+     |_| |_|\__, |_| |_| |_|
+            |___/
+
+             (nym-mixnode - version 1.1.9)
+
+    
+Signing the text "4yRfauFzZnejJhG2FACTVQ7UnYEcFUYw3HzXrmuwLMaR" using your mixnode's Ed25519 identity key...
+The base58-encoded signature on '4yRfauFzZnejJhG2FACTVQ7UnYEcFUYw3HzXrmuwLMaR' is: 4By7EQEMM8BAt6ptxJyeGqpoxHWxeRUhyJ4wMr2x3mXSQD9nvttkvd7tgP1uKu2ktJjB2bLzD1oaZ33d2Wv5eYWp
+~~~
+
+The `--mnemonic` is going to be the mnemonic of the member wanting to join the family: 
 
-The `--from` is going to be the mnemonic of the member wanting to join the family: 
```

Comment:
looks like this line was deleted and not moved anywhere? Its important , so make sure its re-included if its disappeared by accident